modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -687,6 +696,11 @@ public void processOpts() {
 
         additionalProperties.put("lambdaSplitString", new SplitStringLambda());
 
+//        //CM-CUSTOM-START
+//        additionalProperties.put("customEnumRender", new CustomEnumRenderLambda( additionalProperties.get( "discriminatorEnums" ) ) );
+//        additionalProperties.put("removeSurroundingQuotes", new RemoveSurroundingQuotesLambda());
+//        //CM-CUSTOM-END
+
```

Comment:
Is this meant to stay here?

Reply:
Had seen it as well, fixed it in the right commit/branch(i hope to get this merged into the original repo)
but forgot to push...